benchmarks/nemo/nemolite2d/psykal/psyclone_scripts/ocl_transform.py

```diff
@@ -12,8 +14,9 @@ def trans(psy):
     schedule = psy.invokes.get('invoke_0').schedule
 
     # Remove the globals from inside each kernel
```

Comment:
Please mention the purpose of the set_opencl_options call too.

Reply:
Done